app/controllers/registrations_controller.rb

```diff
@@ -0,0 +1,16 @@
+class RegistrationsController < Devise::RegistrationsController
+  skip_before_action :authenticate_user!
+  before_action :configure_permitted_parameters
+
+  def create
+    super { |user| User::Bootstrap.(user) if user.persisted? }
+  end
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:sign_up, keys: %i[name handle])
+  end
+
+  def after_inactive_sign_up_path_for(_resource)
```

Comment:
Also from V2, this ensures that we redirect users to an intermediate page when they haven't confirmed their email yet.

If we sign up via OAuth, this whole flow is bypassed.